calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
```

Comment:
We're defining our valid operators but not actually using them anywhere. With the switch statement you've implemented we're able to identify invalid operators, say may now longer need this.